src/spec-node/devContainersSpecCLI.ts

```diff
@@ -124,14 +135,12 @@ function provisionOptions(y: Argv) {
 	})
 		.check(argv => {
 			const idLabels = (argv['id-label'] && (Array.isArray(argv['id-label']) ? argv['id-label'] : [argv['id-label']])) as string[] | undefined;
+			const isWorkspaceFound = argv['workspace-folder'] || findWorkspaceFolder();
 			if (idLabels?.some(idLabel => !/.+=.+/.test(idLabel))) {
 				throw new Error('Unmatched argument format: id-label must match <name>=<value>');
 			}
-			if (!(argv['workspace-folder'] || argv['id-label'])) {
-				throw new Error('Missing required argument: workspace-folder or id-label');
-			}
-			if (!(argv['workspace-folder'] || argv['override-config'])) {
-				throw new Error('Missing required argument: workspace-folder or override-config');
+			if (!(argv['id-label'] || argv['override-config'] || isWorkspaceFound)) {
+				throw new Error('Missing required argument: workspace-folder or id-label or override-config');
```

Comment:
As `workspace-folder` is no longer a `required` parameter, this error message could be misleading. Instead can we give a similar error (eg. Dev container config not found.) 👇 if workspace is not found? 

https://github.com/devcontainers/cli/blob/efb3ffca505e252a58d8bf4d48da5ffe4becf0e2/src/spec-node/devContainersSpecCLI.ts#L424